src/test/common/test_time.cc

```diff
@@ -176,3 +177,24 @@ TEST(TimePoints, SignedSubtraciton) {
   ASSERT_GT(cmtb - cmta, ceph::signedspan::zero());
   ASSERT_GT((cmtb - cmta).count(), 0);
 }
+
+TEST(Timespan, DefaultTimespanIsZero) {
```

Comment:
@mogeb i still think we should take these three tests out.

Reply:
looks like it's failing, too?

Reply:
Yes, sorry about that. I had made assumptions that are implementation-specific. I fixed them and I'll update the PR soon.